go-controller/pkg/ovn/endpoints.go

```diff
@@ -107,12 +108,26 @@ func (ovn *Controller) AddEndpoints(ep *kapi.Endpoints) error {
 						continue
 					}
 				}
+				ovn.handleExternalIPs(svc, svcPort, ips, targetPort)
 			}
 		}
 	}
 	return nil
 }
 
+func (ovn *Controller) handleExternalIPs(svc *kapi.Service, svcPort kapi.ServicePort, ips []string, targetPort int32) {
+	logrus.Infof("handling external IPs for svc %v", svc.Name)
```

Comment:
We should move it to debug statement to keep consistent with other load-balancer events